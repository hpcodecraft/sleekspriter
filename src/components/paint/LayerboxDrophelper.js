import React from "react";

const LayerBoxDrophelper = () => {
  return (
    <div className="dropzone">
      <div className="dropzone-inner left" />
      <div className="dropzone-inner center" />
      <div className="dropzone-inner right" />
    </div>
  );
};

export default LayerBoxDrophelper;
