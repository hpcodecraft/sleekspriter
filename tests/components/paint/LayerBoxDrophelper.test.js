import LayerboxDrophelper from "components/paint/LayerboxDrophelper";

describe("LayerboxDrophelper", () => {
  it("should render correctly", () => {
    const component = render(<LayerboxDrophelper />);
    expect(component).toMatchSnapshot();
  });
});
