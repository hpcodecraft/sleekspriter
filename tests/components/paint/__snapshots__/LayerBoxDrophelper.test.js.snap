// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayerboxDrophelper should render correctly 1`] = `"<div class=\\"dropzone\\"><div class=\\"dropzone-inner left\\"></div><div class=\\"dropzone-inner center\\"></div><div class=\\"dropzone-inner right\\"></div></div>"`;
